system/bootstrap.php

```diff
@@ -87,6 +87,14 @@
 	define('TESTPATH', realpath($paths->testsDirectory) . DIRECTORY_SEPARATOR);
 }
 
+/**
+ * The path to the views directory
+ */
+if (! defined('VIEWPATH'))
```

Comment:
VIEWPATH wasn't provided on purpose, since views can move around much more often than others. Standard location can be easily accessed via APPPATH.'Views', so not necessary. Please remove.